core/src/test/java/greencity/controller/HabitAssignControllerTest.java

```diff
@@ -90,15 +90,11 @@ void updateAssignByHabitId() throws Exception {
     }
 
     @Test
-    void updateShoppingItemList() throws Exception {
-        HabitAssignPropertiesDto propertiesDto = ModelUtils.getHabitAssignPropertiesDto();
-        Gson gson = new Gson();
-        String json = gson.toJson(propertiesDto);
-        mockMvc.perform(put(habitLink + "/{habitId}/update-user-shopping-item-list", 1L)
-            .content(json)
+    void updateHabitAssignDurationTest() throws Exception {
+        mockMvc.perform(put(habitLink + "/{habitId}/update-habit-duration?duration=15", 1L)
```

Comment:
please change `habitId`  to `habitAssignId`

Reply:
changed